src/app/core/models/attendance-sheets-interface.model.ts

```diff
@@ -0,0 +1,19 @@
+
+export interface NationalHoliday {
```

Comment:
Could you please create a separate interface for the national holiday

Reply:
I added all module interfaces in one file, here the NationalHoliday is used in the attendance module so I added it here